packages/react-app/src/components/global/Header/index.tsx

```diff
@@ -1,89 +1,8 @@
-import React, { useState } from 'react';
```

Comment:
Deleted the menu items from here and moved to previous file. Context API is used to show the customer logo and alt logo text.